R/ttest_Paired.qmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "Paired t-test"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# **Paired t-test in R**
+
+The Paired t-test is used when two samples are naturally correlated. In the Paired t-test, the difference of the means between the two samples is compared to a given number that represents the null hypothesis. For a Paired t-test, the number of observations in each sample must be equal.
+
+In R, a Paired t-test can be performed using the Base R `t.test()` from the **stats** package or the `proc_ttest()` function from the **procs** package.
+
+## Normal Data {#normal}
+
+By default, the R paired t-test functions assume normality in the data and use a classic Student's t-test.
+
+### Data Used
+
+The following data was used in this example.
+
+```{r eval=TRUE, echo = TRUE}
+# Create sample data
+pressure <- tibble::tribble(
+  ~SBPbefore, ~SBPafter,
+  120, 128,   
+  124, 131,   
+  130, 131,   
+  118, 127,
+  140, 132,   
+  128, 125,   
+  140, 141,   
+  135, 137,
+  126, 118,   
+  130, 132,   
+  126, 129,   
+  127, 135
+)
+```
+
+### Base R
+
+#### Code
+
+The following code was used to test the comparison in Base R.
+
+```{r eval=TRUE, echo = TRUE}
+
+  # Perform t-test
+  t.test(pressure$SBPbefore, pressure$SBPafter, paired = TRUE)
+
+```
+
+
+### Procs Package
+
+#### Code
+
+The following code from the **procs** package was used to perform a paired t-test.
+
+```{r eval=FALSE, echo = TRUE}
```

Comment:
Thank you for your updates, can you just make sure all the R code runs? It is important that all the R codes run so that if in the future packages get updated we will be able to see the issues before they get to the website.